### PR TITLE
Update Go, Glide, and container user permissions to work on Darwin 17…

### PR DESCRIPTION
**Pull request by emmetthitz**
Build changes to fix glide/container issues on high sierra.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,11 +13,11 @@ HOST_GOOS = $(shell go env GOOS)
 HOST_GOARCH = $(shell go env GOARCH)
 
 # Versions of external dependencies
-GLIDE_VERSION = v0.11.1
+GLIDE_VERSION = v0.12.3
```
Comment (dkoshkin) [inline Makefile]: Glide v0.13.0 is out

File: Makefile [hunk 2]
```diff
@@ -13,11 +13,11 @@ HOST_GOOS = $(shell go env GOOS)
...
 ANSIBLE_VERSION = 2.3.0.0
 PROVISIONER_VERSION = v1.6.1
 KUBERANG_VERSION = v1.2.2
-GO_VERSION = 1.8.0
+GO_VERSION = 1.8.3
```
Comment (dkoshkin) [inline Makefile]: Go has 1.8.4 and 1.9.1

File: Makefile (hunk 1 above)
Comment (emmetthitz) [reply Makefile]: Good catch.

File: Makefile (hunk 2 above)
Comment (emmetthitz) [reply Makefile]: I'll give go 1.9.1 a shot. It broke some stuff with ansible last time I gave it a shot, but let's see.

**Comment by dkoshkin** [comment]
LGTM!

**Pull request merged**